modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -173,7 +173,11 @@ function FieldBase({
 
 				{displayErrors && errorMessage && !valid && (
 					<span className="form-feedback-group">
-						<div className="form-feedback-item">{errorMessage}</div>
+						<div className="form-feedback-item">
+							<div aria-label={label} aria-live="polite" aria-relevant="additions removals">
```

Comment:
@rijema, why didn't you use `aria-relevant="additions text"`, once you're just changing the text inside the `div`?

Reply:
My intention was to update the live region with the properly changes that I was doing on that specific block of code, based [on this comment](https://issues.liferay.com/browse/PTR-1764?focusedCommentId=2184977&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2184977) received on the PTR that I opened. Therefore, this change mentioned by you is more suited to my initial objective. I do not seek to update the user of a withdrawal of information from the living area, as it would be confused in its usability. I just wanna update him about an addition of a text in this area.

I added a new commit making changes said here. Let me know if you want me to update this pull request with a new commit or if I have to squash in some of these.